### PR TITLE
1409 adds pagination to view

### PR DESCRIPTION
**Pull request by Regan831**

### CODE REVIEW

**Comment by potato620** [comment]
lgtm

**Comment by staticc** [comment]
lgtm

File: app/index.html [hunk 1]
```diff
@@ -28,13 +28,21 @@
                     </div>
                     <div class="search-container">
                         <input type="text" ng-model="search" class="search-input" placeholder="Search">
-                        <i class="fa fa-search search-icon"></i>
+                        <i class="fa fa-search search-icon" ng-click="curPage = 0; loadPage(pageSize, 0, search)"></i>
```
Comment (sarahleeashraf) [inline app/index.html]: is this the right place to put the function call? Seems odd to have it in markup.

Comment (Regan831) [inline app/index.html]: I'm not sure. There were other functions being called in that way, so I thought to do the search and change page buttons the same way. Is there a better way to do it?

Comment (sarahleeashraf) [inline app/index.html]: Definitely seems like it is not easily testable. Have we looked into how to write tests for angular yet?

Comment (Regan831) [inline app/index.html]: This function comes from the controller 'mainCtrl'. Looks like we can test functions from a controller like this, https://gist.github.com/blesh/8846528

**Pull request closed without merging**